Sources/SotoCodeGenerator/AwsService+shapes.swift

```diff
@@ -119,7 +119,8 @@ extension AwsService {
                     shapeOptions.append("allowStreaming")
                     if !payload.hasTrait(type: RequiresLengthTrait.self),
                        let operationShape = operationShape,
-                       operationShape.hasTrait(type: AwsAuthUnsignedPayloadTrait.self) {
+                       operationShape.hasTrait(type: AwsAuthUnsignedPayloadTrait.self)
+                    {
```

Comment:
Was this meant to be changed?
```suggestion
                       operationShape.hasTrait(type: AwsAuthUnsignedPayloadTrait.self) {
```

Reply:
Ah fair enough